app/pages/project/affinity/AffinityPage.tsx

```diff
@@ -98,8 +103,54 @@ const staticCols = [
 
 export default function AffinityPage() {
   const { project } = useProjectSelector()
-  const columns = useMemo(
-    () => [
+  const { data } = usePrefetchedQuery(antiAffinityGroupList({ project }).optionsFn())
```

Comment:
Small thing, but because we're not passing these calls to `useQueryTable` because there's no pagination, you can use `apiq` instead of `getListQFn` to make this helper, and then you can drop the `.optionsFn()` everywhere.

Reply:
I made the change since it's trivial.